schema/authorisation.js

```diff
@@ -9,18 +9,18 @@ class Authorisation extends BaseModel {
   static get jsonSchema() {
     return {
       type: 'object',
-      required: ['type', 'method', 'description'],
+      required: ['type', 'method', 'description', 'establishmentId'],
```

Comment:
I missed `establishmentId` as a required field from authorisation (having a memory failure if this was intentional or not?)

Reply:
It's definitely required. Good catch.

Reply:
Yeah establishmentId should be required